package.json

```diff
@@ -30,12 +30,13 @@
   },
   "homepage": "https://github.com/transitmatters/new-train-tracker#readme",
   "dependencies": {
-    "@types/react": "^18.0.1",
+    "@types/react": "^17.0.0",
     "@types/react-dom": "^18.0.0",
+    "acorn": "^8.8.0",
     "bezier-js": "^2.5.1",
     "classnames": "^2.2.6",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
```

Comment:
need for peer dep 

```
npm WARN acorn-import-assertions@1.8.0 requires a peer of acorn@^8 but none is installed. You must install peer dependencies yourself.
npm WARN react-favicon@1.0.0 requires a peer of react@^16.0.0 || ^17.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN reakit@1.3.11 requires a peer of react@^16.8.0 || ^17.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN reakit-system@0.15.2 requires a peer of react@^16.8.0 || ^17.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN reakit-utils@0.15.2 requires a peer of react@^16.8.0 || ^17.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN reakit-warning@0.6.2 requires a peer of react@^16.8.0 || ^17.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN react-dom@17.0.2 requires a peer of react@17.0.2 but none is installed. You must install peer dependencies yourself.
```

My understanding was this is the reason the listing kept failing. Someone more familiar with the GitHub workflow correct me if I'm wrong.

Reply:
@mathcolo any issues with downgrading this to react 17?

Reply:
I don't think so, but I don't quite understand what the issue with 18 is. Can we upgrade other packages so we don't have to go backwards?